tools/Environments/DevHome.Environments/Helpers/DataExtractor.cs

```diff
@@ -85,12 +85,12 @@ public static List<OperationsViewModel> FillLaunchButtonOperations(ComputeSystem
 
         if (supportedOperations.HasFlag(ComputeSystemOperations.Resume))
         {
-            operations.Add(new OperationsViewModel("Resume", "\uF2C6", computeSystem.ResumeAsync));
+            operations.Add(new OperationsViewModel("Resume", "\uE768", computeSystem.ResumeAsync));
         }

```

Comment:
We should make an issue on GitHub about adding localization to the operation text

Reply:
Noted.